kie-wb-common-stunner/kie-wb-common-stunner-sets/kie-wb-common-stunner-case-mgmt/kie-wb-common-stunner-case-mgmt-client/src/main/java/org/kie/workbench/common/stunner/cm/client/wires/CaseManagementContainmentControl.java

```diff
@@ -174,11 +176,13 @@ public void destroy() {
     }
 
     private void restore(final CaseManagementShapeView ghost) {
-        final WiresContainer originalParent = state.getOriginalParent().get();
-        final int originalIndex = state.getOriginalIndex().get();
-        restore(ghost,
-                originalParent,
-                originalIndex);
+        if (state.getOriginalParent().isPresent() && state.getOriginalIndex().isPresent()) {
```

Comment:
just a suggestion on this block:

```suggestion
state.getOriginalIndex()
    .ifPresent(originalIndex -> state.getOriginalParent()
        .ifPresent(originalParent -> restore(ghost, originalParent, originalIndex)));
```